src/ufm/fine_tuning.py

```diff
@@ -80,12 +120,17 @@ def tokenize_function(examples):
         trainer.train()
 
         logger.info("Evaluating model...")
-        eval_results = trainer.evaluate()
-        
-        return eval_results['eval_loss'] #validation loss for fine-tuning
+        #eval_results = trainer.evaluate()
+        trainer.save_model()
```

Comment:
Does this save it to disk? We probably don't want this for now, just to keep it in memory. Would recommend either putting this within an if check for a new config flag, or else removing.